RSSReader/View/FeedViewController/FeedViewController.m

```diff
@@ -162,6 +165,32 @@ - (void)refreshTableViewCell:(int)index {
     } completion:nil];
 }
 
+- (void)motionEnded:(UIEventSubtype)motion withEvent:(UIEvent *)event {
+    if (motion == UIEventSubtypeMotionShake) {
+        [self showRefreshAlert];
+    }
+}
+
+- (void)showRefreshAlert {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString(kReloadAlertMessage, nil)
```

Comment:
This functionality should be in the extension

Reply:
added